test/parallel/test-whatwg-url-tojson.js

```diff
@@ -4,13 +4,11 @@ const common = require('../common');
 const URL = require('url').URL;
 const { test, assert_equals } = common.WPT;
 
-/* eslint-disable */
 /* WPT Refs:
    https://github.com/w3c/web-platform-tests/blob/02585db/url/url-tojson.html
    License: http://www.w3.org/Consortium/Legal/2008/04-testsuite-copyright.html
 */
 test(() => {
-  const a = new URL("https://example.com/")
-  assert_equals(JSON.stringify(a), "\"https://example.com/\"")
-})
-/* eslint-enable */
+  const a = new URL('https://example.com/');
```

Comment:
This should probably be reverted..also, the test suites are dual-licensed. I am not sure which license we are using but generally:

> performance claims can only be made against unaltered tests